wallet-new/server/Cardano/Wallet/API/V1/LegacyHandlers/Wallets.hs

```diff
@@ -107,4 +107,4 @@ updateWallet
     :: WalletId
     -> WalletUpdate
     -> MonadV1 (WalletResponse Wallet)
-updateWallet _ _ = single <$> (liftIO $ generate arbitrary)
+updateWallet _ _ = single <$> liftIO (generate arbitrary) -- error "Unimplemented: tracked in CSL-2450"
```

Comment:
yes, I would rather throw an explicit error here then prettend everything works (especially as we are going to roll out soon). Good catch